drivers/driver.go

```diff
@@ -34,6 +35,8 @@ var (
 	ErrPrerequisites = errors.New("prerequisites for driver not satisfied (wrong filesystem?)")
 	// ErrIncompatibleFS returned when file system is not supported.
 	ErrIncompatibleFS = fmt.Errorf("backing file system is unsupported for this graph driver")
+	// ErrLayerUnknown returned when the specified layer is unknown by the driver.
+	ErrLayerUnknown = fmt.Errorf("unknown layer")
```

Comment:
nit: use `errors.New` as there's no `%`-style formatting in here.

Feel free to also fix the definition right above (in a separate commit).